views/legislation-page.js

```diff
@@ -231,6 +231,11 @@ const votePositionClass = (position) => {
   return ''
 }
 
+const subjectUrl = (url, policy_area) => {
```

Comment:
The logic in this function looks brittle. 
Why depend on `terms`, `legislature`, unrelated params?

Reply:
Probably safer to re-use the related [`makeQuery` function](https://github.com/voteliquid/liquid.us/blob/db54879f7988ee801ff27b8319734263e72aec6b/views/legislation-page.js#L303) in that file, adding this new `policy_area` as a key to the `query` param.

Reply:
I tried the makeQuery function, but it ends up putting order=policy_area into the url